### DISCUSSION TITLE
change todo and note index pages to two column mode

### DISCUSSION THREAD

**Discussion by ehula** [kudrykv/latex-yearly-planner#138] [General]
Is it possible for you to change the Todos and Notes Index pages? The current format of a 10x10 grid of boxes is not ideal. I would prefer to write a title for each todo page and note page on the corresponding index page, but the box is so small that it is difficult to write anything meaningful that is readable. Can you instead show two columns of notes/todos on the index pages, with a single line for each item. I believe you use to do soemthing similar, but with a single column. I'd be happy with two columns with 38ish items in each column.

**Comment by kudrykv** [comment]
You might want to poke the WIP branch -- https://github.com/kudrykv/latex-yearly-planner/tree/rubify2
Even though it's in progress, it finally solidifies. Config file should also be far more pleasant in comparison to what exists today.
Planner with notes example -- 
[index.pdf](https://github.com/user-attachments/files/17642623/index.pdf)

Shout out if you need help or if you need a planner generated for you. In latter case, I'd need to know details you need (physical device screen size, where your PDF viewer has a toolbar and how much space does it take, dotted vs lined, 24 hours or AMPM, weekdays start on Sunday or Monday).

"breadcrumb" header is not yet supported there.

Within the existing code -- latest code creates a "numbered list" table, with single column, yes.

Or do you mean merging todos and notes together, putting index lists side by side? If so, that is something that needs implementation, but I'm not sure how I feel about it

**Comment by ehula** [comment]
I have a Boox Note Air 3C. I have been using rm2.mos.lined.default.ampm.dailycal.2024.pdf. It seems to fit well, and the empty space in the upper right corner makes room for Boox's bookmark button.

The comments I made above were based on the screenshots on this page: https://github.com/kudrykv/latex-yearly-planner/discussions/9.
I now realize that these screenshots are old and do not reflect the current pdfs. So I downloaded this one: https://github.com/user-attachments/files/16644971/rm2.mos.lined.default.ampm.dailycal.2025.pdf

Here are two things I would like to see changed:
1. The Notes Index page is a list of 114 items, 38 per page, three pages total. I don't think one needs all that room to write a description for each note. I would love to see the Notes Index page have two columns; 1-38 and 39-76. You could add a second page with 77-114 and 115-152.
2. Add back the Todos Index and Todos pages. Format the Todos Index pages in a similar way to the Notes Index page described above.

If you could generate this planner for me, that would be awesome!

**Reply by kudrykv** [reply]
I need to update screenshots indeed!

> the empty space in the upper right corner makes room for Boox's bookmark button

I don't have this part yet. I still need to catch up on some bits, and that one of them. Hopefully I can catch up with all the things that the current implementation has before 2025. I'll note the request
